BuildResidentialHPXML/measure.rb

```diff
@@ -3970,14 +4039,55 @@ def self.set_heating_systems(hpxml, runner, args)
       heating_efficiency_percent = args[:heating_system_heating_efficiency_percent]
     end
 
+    fraction_heat_load_served = args[:heating_system_fraction_heat_load_served]
+    fraction_heat_load_served_2 = args[:heating_system_fraction_heat_load_served_2]
+    fraction_heat_load_served -= fraction_heat_load_served_2
+
     hpxml.heating_systems.add(id: 'HeatingSystem',
                               heating_system_type: heating_system_type,
                               heating_system_fuel: heating_system_fuel,
                               heating_capacity: heating_capacity,
-                              fraction_heat_load_served: args[:heating_system_fraction_heat_load_served],
+                              fraction_heat_load_served: fraction_heat_load_served,
                               electric_auxiliary_energy: electric_auxiliary_energy,
                               heating_efficiency_afue: heating_efficiency_afue,
                               heating_efficiency_percent: heating_efficiency_percent)
+
+    heating_system_type_2 = args[:heating_system_type_2]
+
+    return if heating_system_type_2 == 'none'
+
+    heating_capacity_2 = args[:heating_system_heating_capacity_2]
+    if heating_capacity_2 == Constants.Auto
+      heating_capacity_2 = -1
```

Comment:
You can just set all autosized heating capacities to `nil` now instead of -1 (though -1 still works for backwards compatibility). Excluding the element is now preferred to setting it as -1.